plugins/modules/zos_job_submit.py

```diff
@@ -513,7 +515,7 @@
 POLLING_INTERVAL = 1
 POLLING_COUNT = 60
 
-JOB_COMPLETION_MESSAGES = ["CC", "ABEND", "SEC ERROR"]
+JOB_COMPLETION_MESSAGES = ["CC", "ABEND", "SEC", "JCL ERROR"]
```

Comment:
That looks like a mistake on my part... I may have thought the 'error' was for jcl, and split the string.  My concern with the stand-alone word 'error' is that this list is used to find out if there was a completion to the job, but it could also come up as a single work in the source dump or a dd reply, and be mis-interpreted.  For safety, I'll add error back to "sec".

Reply:
Thank you Rich